### ISSUE TITLE
The Tutorial doesn't match with the current version of Mingus?

### ISSUE THREAD

**Issue by ianschwartz** [bspaans/python-mingus#48]
core.diatonic module non-existant in the current version of mingus. Completely missing. Makes following the tutorial a little hard.

**Comment by oxygen-dioxide** [comment]
http://bspaans.github.io/python-mingus/doc/wiki/tutorialNote.html#minor-and-major-conversions `notes.to_minor` and `notes.to_major` is also missing

**Comment by kcrouch** [comment]
In case someone else stumbles here, looks like someone did some work around that at https://github.com/bspaans/python-mingus/pull/29/files

**Comment by tomsocoolsky** [comment]
Exactly, I think core.diatonic has been changed to: core.keys. Is there a chance to change it in the tutorial? As a newbie I am happy to help here :)
